### DISCUSSION TITLE
Check ssh config file instead of selecting key

### DISCUSSION THREAD

**Discussion by anvanza** [spatie/ray#817] [Ideas]
I'm using the remote server a lot. It allows me to debug issues on production servers quickly.

We've moved to the 1password ssh functionality. With this you're not saving the ssh key on your local machine anymore.
It allows us to rotate SSH keys a lot quicker.

Currently, Ray is the only tool that doesn't work with 1 Password.
[1 password compatibility ](https://developer.1password.com/docs/ssh/agent/compatibility/)

**Comment by alexevladgabriel** [comment]
That would be great @freekmurze, it's a pain in the ass.

**Comment by ruben-haegeman** [comment]
In light of recent supply chain attack I think it would be great if we could protect our SSH keys better by using 1Password
